app/jobs/request_verification_code_for_login_job.rb

```diff
@@ -31,12 +31,20 @@ def perform(email_address: nil, phone_number: nil, locale:, visitor_id:, service
           phone_number: phone_number,
           locale: locale,
           visitor_id: visitor_id,
-          service_type: multi_tenant_service.service_type
+          service_type: multi_tenant_service.service_type_or_parent
         )
       else
-        service_name = multi_tenant_service.service_name
         url = multi_tenant_service.url(locale: locale)
-        body = service_name == "GetCTC" ? I18n.t("verification_code_sms.no_match_ctc", url: url, locale: locale) : I18n.t("verification_code_sms.no_match_gyr", url: url, locale: locale)
+        body = case service_type
+               when :ctc
+                 I18n.t("verification_code_sms.no_match_ctc", url: url, locale: locale)
+               when :gyr
+                 I18n.t("verification_code_sms.no_match_gyr", url: url, locale: locale)
+               when :statefile_az
+                 I18n.t("state_file.intake_logins.no_match_sms", url: "#{url}/az/questions/landing-page", locale: locale)
+               when :statefile_ny
+                 I18n.t("state_file.intake_logins.no_match_sms", url: "#{url}/ny/questions/landing-page", locale: locale)
+               end
```

Comment:
Non-blocking:
- How much can/should we pull into methods on multi_tenant_service? 
- Can we use a better method for determining the URL?

Reply:
I think it makes sense to use this landing page as the .url value for state file. I would also find it helpful to have a comment saying that this is a "start page" or to rename the method to give more context about how it is expected to be used.